internal/services/network/bastion_host_resource_test.go

```diff
@@ -27,6 +27,7 @@ func TestAccBastionHost_basic(t *testing.T) {
 			Config: r.basic(data),
 			Check: acceptance.ComposeTestCheckFunc(
 				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("zones.#").HasValue("3"),
```

Comment:
Can we remove this check since it's not valid for the basic test anymore

Reply:
Good catch, thank you :)